example-react-app/src/app/associationM2M-management/AssociationM2MEditor.tsx

```diff
@@ -74,10 +74,11 @@ const AssociationM2MEditor = observer(
       loadQuery: LOAD_SCR_ASSOCIATIONM2MTESTENTITY,
       upsertMutation: UPSERT_SCR_ASSOCIATIONM2MTESTENTITY,
       entityName: ENTITY_NAME,
-      routingPath: ROUTING_PATH,
-      onCommit,
       entityInstance,
-      persistEntityCallbacks: useEntityPersistCallbacks(),
+      callbacks: useEntityEditorCallbacks({
+        routingPath: ROUTING_PATH,
+        onCommit
```

Comment:
It think it's better to spread `onCommit`:

```
callbacks: {
  ...useEntityEditorCallbacks(...),
  onCommit
}
```

I.e. `useEntityEditorCallbacks` will return default implementation of `onCommit`, but we replace it with custom `onCommit` if such one has been passed in Editor props.